ac_s.lua

```diff
@@ -851,6 +865,22 @@ Citizen.CreateThread(function()
 		end
 	end)
 
+	AddEventHandler("clearPedTasksEvent", function(source, data)
```

Comment:
wouldnt this also trigger if tasks of normal peds were cleared by a script? I would narrow it down and double check if the source of the event is not also the ped its being executed on, and making sure the ped is a player.